tests/cross/src/test/scala/tests/highlight/DocumentHighlightSuite.scala

```diff
@@ -655,4 +655,14 @@ class DocumentHighlightSuite extends BaseDocumentHighlightSuite {
        |}""".stripMargin,
   )
 
+  check(
+    "named-param",
```

Comment:
Is this fixed in this PR? IF so could you maybe add it in a separate PR?

Reply:
Yes, here 
https://github.com/scalameta/metals/pull/4542/files#diff-ccf6318d7fdc8644e313a74a01d410c3d7e7c09750f4ff260b902ee7ab856ddaR119-R137

 (although I wouldn’t call it a fix yet, maybe a workaround 😅)
Sure, I can move it to separate PR, good idea

Reply:
Moved it here https://github.com/scalameta/metals/pull/4599